src/Pos/Txp/Core/Types.hs

```diff
@@ -152,10 +152,16 @@ instance Buildable TxOut where
     build TxOut {..} =
         bprint ("TxOut "%coinF%" -> "%build) txOutValue txOutAddress
 
-type TxOutAux = (TxOut, [(StakeholderId, Coin)])
+-- | Transaction output and auxilary data corresponding to it.
+-- [CSL-366] Add more data.
```

Comment:
You can assign this type of refactoring to somebody else ;)
I can perform this refactoring, should be no problems.
> Why is there TxOutAux in binary-protocols?
When I reviewed code I was convinced that `TxOutAux` is what sent over network. Maybe I'm wrong. But I don't know whom to ask.